libr/debug/p/native/maps/windows_maps.c

```diff
@@ -101,7 +104,7 @@ R_API RList *r_w32_dbg_modules(RDebug *dbg) {
 		}
 	} while (Module32Next (h_mod_snap, &me));
 err_w32_dbg_modules:
-	if (h_mod_snap) {
+	if (h_mod_snap && h_mod_snap != INVALID_HANDLE_VALUE) {
```

Comment:
@GustavoLCR, I think the only check `h_mod_snap != INVALID_HANDLE_VALUE` should remain here according to the API